### ISSUE TITLE
TitleBar/MenuBar Settings

### ISSUE THREAD

**Issue by AutumnWohlen** [electron/electron#17532]
Good day
I like Electron because it has a good code implementation.
BrowserWindow has a lot of nice options for customization
[https://electronjs.org/docs/api/browser-window](url)

But I hope you will add some simple options for TitleBar customization
For some easy applications, change a color bar is more than enough. For more advanced applications I use index.html file with custom navbar settings and styles.

Proposed Solution
You have already added -titleBarStyle- String and it's really fine for me and my users on MacOS.
So I hope you will add some more options in near future.
My proposition is:
titleBarColor: String (optional) use HEXcolor (#80FFFFFF), // TitleBar color. It is really, what I wish for my users
titleBarHeight: Integer (optional) use values (50), // TitleBar height
titleBarTextColor: String (optional) use HEXcolor (#80FFFFFF), // Text color in TitleBar
titleBarElementsColor: String (optional) use HEXcolor (#80FFFFFF) // Some buttons elements color in TitleBar

With Great respect
A.W.

**Comment by emilbayes** [comment]
Have you seen these docs? https://github.com/electron/electron/blob/master/docs/api/frameless-window.md 

That's how Atom and VS Code does it AFAIK

**Comment by AutumnWohlen** [comment]
I use Frameless Window for custom UI, but I didn't notice any options for changing color of the BrowserWindow when a frame option is set "true" (frame: true - is default). On MacOS you can use "titleBarStyle: hidden" and it looks nice but for Linux and Windows I need something else.

**Comment by fabiospampinato** [comment]
I don't think this is quite implementable for native titlebars, as it would depend on what each underlying UI framework can do, but you can do whatever you want already when reimplementing the titlebar yourself.
